src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -215,7 +219,7 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
     val pid = intelliJ.probe.pid()
     intelliJ.probe.screenshot("exception")
     val threadStackTrace: String = Process("jstack " + pid) !!
-    val artifactDirectory = "/tmp/ide-probe/screenshots/uiTest" + intelliJVersion.build + "/artifacts"
+    val artifactDirectory = "/build/UIException/uiTest" + intelliJVersion.build + "/artifacts"
```

Comment:
Actually... I how about using Gradle's `build/` folder in the current directory?
```suggestion
    val artifactDirectory = "build/screenshots/uiTest" + intelliJVersion.build + "/artifacts"
```

Reply:
but this` /build/UIException/uiTest` does not refer to build folder in root directory ? (because screenshots are saving there - I pass `/build `to `MY_CI_SCREENSHOTS_DIR`)

Reply:
Hmmm with a leading slash? I'd expect it to refer to an absolute path 🤔

Reply:
Or even better... pass the path to `build/` directory in the project in `MY_CI_SCREENSHOTS_DIR` (plus `screenshots/` folder)!

See `systemProperty 'ui-test.plugin.path'` for a similar trick, for passing Gradle paths to the otherwise Gradle-agnostic tests